styles/style.css

```diff
@@ -6,3 +6,117 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    margin: 0;
+    top: 0;
+}
+
+header img {
+    width: 150px;
+}
+
+.head {
+    margin: 5rem;
+    display: flex;
+    justify-content: space-around;
+}
+
+nav {
+    width: 20%;
+    justify-content: flex-end;
+}
+
+nav li {
+    list-style: none;
+    padding: 5px;
+}
+
+nav ul {
+    display: flex;
+}
+
+nav a {
+    text-decoration: none;
+    color: #1A1A1A;
+}
+
+.landing {
+    background-image: url(/images/landing.jpg);
+    color: white;
+    background-size: cover;
+    height: 500px;
+    font-size: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-bottom: 50px;
+}
+
+
+article h1, p {
+    text-align: center;
+}
+
+.middle h2, hr {
+    margin: 0 auto;
+    text-align: center;
+    width: 300px;
+    border-bottom: 2px solid #00B172;
+}
+
+.icons {
+    text-align: center;
+    display: flex;
+    margin-top: 50px;
```

Comment:
here I also would suggest to use justify-content:space-around, to spread the elements evenly